stat/sampleuv/withoutreplacement.go

```diff
@@ -0,0 +1,54 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package sampleuv
+
+import (
+	"math/rand"
+	"sort"
+)
+
+// WithoutReplacement samples len(idxs) integers from [0, max) without replacement.
```

Comment:
I think that you mean `n` where you say `max` (also in the function body comment).